app/components/shared/searchable_collection_component/searchable_collection_component.scss

```diff
@@ -12,7 +12,7 @@
 
   &.collection-component--border {
     background-color: #fff;
-    border: 5px solid #f3f2f1;
+    border: 2px solid #b1b4b6;
```

Comment:
what's the `govuk-colour`?

Reply:
ah i couldnt use the gov color before i relocated the scss file yesterday, but now i can...